x.py

```diff
@@ -116,14 +119,43 @@ def build(dir: str, jobs: int, ghproxy: bool, ninja: bool, unittest: bool, compi
         target.append("unittest")
     run(cmake, "--build", ".", f"-j{jobs}", "-t", *target, verbose=True, cwd=dir)
 
-def cpplint() -> None:
-    command = find_command("cpplint", msg="cpplint is required")
-    options = ["--linelength=120", "--filter=-build/include_subdir,-legal/copyright,-build/c++11"]
-    sources = [*glob("src/*.h"), *glob("src/*.cc")]
-    run(command, *options, *sources, verbose=True)
+def get_source_files() -> List[str]:
+    return [
+        *glob("src/**/*.h", recursive=True),
+        *glob("src/**/*.cc", recursive=True),
+        *glob("tests/cppunit/**/*.h", recursive=True),
+        *glob("tests/cppunit/**/*.cc", recursive=True),
+        *glob("tools/kvrocks2redis/**/*.h", recursive=True),
+        *glob("tools/kvrocks2redis/**/*.cc", recursive=True),
+    ]
+
+def check_format(clang_format_path: str, fix: bool = False) -> None:
```

Comment:
```suggestion
def clang_format(clang_format_path: str, fix: bool = False) -> None:
```

... also its references.

Reply:
done